LineChart.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+import Charts
+
+struct LineChart: View {
+    // Sample data
+    let data: [ChartData] = [
```

Comment:
perhaps we should take data in as a variable? not sure where the var would be originated, but then we could call the component with the data passed in

Reply:
yes that was my plan eventually. I just hardcoded it here since we dont have any data rn